WebDriverManager/DriverConfigs/Impl/ChromeConfig.cs

```diff
@@ -102,7 +102,14 @@ private string GetRawBrowserVersion()
 
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
             {
-                return RegistryHelper.GetInstalledBrowserVersionLinux("google-chrome", "--product-version");
+                try
```

Comment:
I'm not sure about try/catch construction here, maybe it will be better to set chrome check result into variable and in case of empty previous check result return chromium result then?